public/app/features/dashboard-scene/panel-edit/PanelEditor.tsx

```diff
@@ -138,7 +138,7 @@ export class PanelEditor extends SceneObjectBase<PanelEditorState> {
       width: repeatDirection === 'h' ? 24 : gridItem.state.width,
       height: gridItem.state.height,
       itemHeight: gridItem.state.height,
-      source: panelManager.state.panel.clone(),
+      source: panelManager.state.panel.clone({ $data: panelManager.state.$data?.clone() }),
```

Comment:
maybe we can extract this to function on VizPanelManager as it is repeated  so many times

Reply:
Good idea